app/assets/images/payment_icons/dankort.svg

```diff
@@ -1 +1 @@
-<svg viewBox="0 0 38 24" xmlns="http://www.w3.org/2000/svg" width="38" height="24" role="img" aria-labelledby="pi-dankort"><title id="pi-dankort">Dankort</title><path d="M3 0C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3H3z" fill="#D8232A"/><path d="M27.144 6c-.15.062-3.395 5.163-3.395 5.163L24 6h-7s-.357 3.054-.445 4.482C16.22 8.85 13.635 6 10.04 6H0v12h10.527c1.953 0 3.778-1.236 4.892-3.028.4-.657.71-1.338.98-2.04-.135 1.632-.202 3.413-.38 5.068 2.383-.023 7.373 0 7.373 0 .044-2.04.29-6.293.29-6.293L27.86 18H37c-1.746-2.037-3.537-4.44-5.076-6.627l.023-.095C33.56 9.494 35.214 7.6 37 6h-9.856zM5.128 14v-3.99c1.33.1 4.803-.066 4.803 1.995 0 2.06-3.477 1.933-4.802 1.995z" fill="#fff"/><path d="M.5 6.5v11" stroke="#e5e5e5"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="38" height="24" viewBox="0 0 38 24" role="img" aria-labelledby="pi-dankort"><title id="pi-dankort">Dankort</title><g fill-rule="evenodd" clip-rule="evenodd" id="pi-dankort-background_dankort"><path fill="#FFF" d="M36 24H2c-1.333 0-2-.7-2-2V2C0 .667.7 0 2 0h34c1.333 0 2 .7 2 2v20c0 1.333-.667 2-2 2z"/><path fill="none" stroke="#000" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="3" stroke-opacity=".07" d="M36 24c1.333 0 2-.7 2-2V2c0-1.333-.7-2-2-2H2C.7 0 0 .7 0 2v20c0 1.333.7 2 2 2h34z"/></g><path fill-rule="evenodd" clip-rule="evenodd" fill="#E31F28" d="M14.7 10.6c.3.267.384.633.25 1.1-.167.633-.383 1.066-.65 1.3-.333.267-.833.4-1.5.4H8.2l1.15-3.15h4.199c.535 0 .918.117 1.151.35zm5.7 1.85L23.95 17h6.95a8.363 8.363 0 01-2.75 2.2c-1.1.533-2.284.8-3.55.8H11.601c-1.2 0-2.351-.267-3.45-.8A7.727 7.727 0 015.4 17h8.1c2.033 0 3.601-.384 4.7-1.15 1.133-.7 1.866-1.833 2.2-3.4zM24.6 4c1.2 0 2.334.267 3.4.8a7.924 7.924 0 012.7 2h-6.75l-3.4 3.5c-.133-1.233-.733-2.133-1.8-2.7-.967-.533-2.316-.8-4.05-.8H5.55a7.82 7.82 0 012.75-2c1.067-.533 2.167-.8 3.301-.8H24.6zM31 7.25c.5.667.9 1.4 1.2 2.2.267.833.4 1.683.4 2.55 0 .833-.133 1.65-.399 2.45-.2.767-.55 1.45-1.05 2.05l-4.25-5.1L31 7.25z" id="pi-dankort-icon_dankort"/></svg>
```

Comment:
Do we need the `id` attributes that are not used?

Also the border needs to follow like other icons: [video demonstration](https://screenshot.click/05-38-bz5z6-aqu3p.mp4)

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="38" height="24" viewBox="0 0 38 24" role="img" aria-labelledby="pi-dankort"><title id="pi-dankort">Dankort</title><g fill-rule="evenodd" clip-rule="evenodd"><path fill="#FFF" d="M36 24H2c-1.333 0-2-.7-2-2V2C0 .667.7 0 2 0h34c1.333 0 2 .7 2 2v20c0 1.333-.667 2-2 2z"/><path fill="none" stroke="#000" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="3" stroke-opacity=".07" d="M36 24c1.333 0 2-.7 2-2V2c0-1.333-.7-2-2-2H2C.7 0 0 .7 0 2v20c0 1.333.7 2 2 2h34z"/></g><path fill-rule="evenodd" clip-rule="evenodd" fill="#E31F28" d="M14.7 10.6c.3.267.384.633.25 1.1-.167.633-.383 1.066-.65 1.3-.333.267-.833.4-1.5.4H8.2l1.15-3.15h4.199c.535 0 .918.117 1.151.35zm5.7 1.85L23.95 17h6.95a8.363 8.363 0 01-2.75 2.2c-1.1.533-2.284.8-3.55.8H11.601c-1.2 0-2.351-.267-3.45-.8A7.727 7.727 0 015.4 17h8.1c2.033 0 3.601-.384 4.7-1.15 1.133-.7 1.866-1.833 2.2-3.4zM24.6 4c1.2 0 2.334.267 3.4.8a7.924 7.924 0 012.7 2h-6.75l-3.4 3.5c-.133-1.233-.733-2.133-1.8-2.7-.967-.533-2.316-.8-4.05-.8H5.55a7.82 7.82 0 012.75-2c1.067-.533 2.167-.8 3.301-.8H24.6zM31 7.25c.5.667.9 1.4 1.2 2.2.267.833.4 1.683.4 2.55 0 .833-.133 1.65-.399 2.45-.2.767-.55 1.45-1.05 2.05l-4.25-5.1L31 7.25z"/></svg>
```